src/rgw/setup/templates/rgw_core_logrotate.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# script is used to delete the old rgw core files in /etc/cortx/log/rgw/<machine-id>/rgw_debug directory
+# script will retain latest 5 core file and remove rest of core files.
+# argument1: <number of latest rgw core files to retain>
+# Default number of latest log files is 5
+# ./rgw_core_logrotate.sh -n 5
+
+usage() { echo "Usage: bash $(basename "$0")[--help|-h]
+                   [-n maxRgwCoreFileCount]
+Retain recent generated rgw core files of given count and remove rest of core files.
+
+where:
+-n            number of latest rgw core files to retain (Default count for core files is 5)
+--help|-h     display this help and exit" 1>&2; exit 1; }
+
+# max rgw core files count in each rgw log directory
+core_files_max_count=5
```

Comment:
5 cores by default would be high, given one core file size can have atleast 500MB size, lets keep 2

Reply:
okay.will update it

Reply:
done. added default value as 2